code_challenges/code_wars/increment_string.py

```diff
@@ -0,0 +1,31 @@
+def increment_string(characters):
+    if characters == "":
+        return "1"
+    if characters[-1].isalpha():
+        characters += "1"
+    else:
+        numbers = ""
+        stop = None
+        for i in range(len(characters) - 1, -1, -1):
+            if characters[i].isdigit():
+                numbers += characters[i]
+                stop = i
+            else:
+                break
+
+        characters = characters[:stop]
```

Comment:
Can you think of a way to replace this with `re` like here https://github.com/aepaysinger/code-challenges/blob/main/code_challenges/code_wars/football_cards.py#L14